docs/minishift.md

```diff
@@ -24,17 +24,16 @@ Minishift is a command-line tool that provisions and manages single-node OpenShi
 ```
 
 ### SEE ALSO
-* [minishift config](minishift_config.md)	 - Modifies Minishift configuration properties.
-* [minishift console](minishift_console.md)	 - Opens or displays the OpenShift Web Console URL.
-* [minishift delete](minishift_delete.md)	 - Deletes the Minishift VM.
-* [minishift docker-env](minishift_docker-env.md)	 - Sets Docker environment variables.
-* [minishift get-openshift-versions](minishift_get-openshift-versions.md)	 - Gets the list of OpenShift versions available for Minishift.
-* [minishift ip](minishift_ip.md)	 - Gets the IP address of the running cluster.
-* [minishift logs](minishift_logs.md)	 - Gets the logs of the running OpenShift cluster.
-* [minishift service](minishift_service.md)	 - Prints the URL for the specified service to the console.
-* [minishift ssh](minishift_ssh.md)	 - Log in to or run a command on a Minishift VM with SSH.
-* [minishift start](minishift_start.md)	 - Starts a local OpenShift cluster.
-* [minishift status](minishift_status.md)	 - Gets the status of the local OpenShift cluster.
-* [minishift stop](minishift_stop.md)	 - Stops the running local OpenShift cluster.
-* [minishift version](minishift_version.md)	 - Gets the version of Minishift.
-
+* [minishift config](./minishift_config.md)	 - Modifies Minishift configuration properties.
```

Comment:
This is really a generated page. It won't help to edit this by hand. I will drop this when merging